sdk/python/tests/integration/registration/test_universal_registry.py

```diff
@@ -228,10 +229,15 @@ def handler(request):
 def mock_remote_registry():
     fd, registry_path = mkstemp()
     registry_config = RegistryConfig(path=registry_path, cache_ttl_seconds=600)
-    proxied_registry = Registry("project", registry_config, None)
+    proxied_registry = Registry(
+        "project", registry_config, None, auth_config=NoAuthConfig()
+    )
 
     registry = RemoteRegistry(
-        registry_config=RemoteRegistryConfig(path=""), project=None, repo_path=None
+        registry_config=RemoteRegistryConfig(path=""),
+        project=None,
+        repo_path=None,
+        auth_config=NoAuthConfig(),
```

Comment:
better to keep it as default value so we need to pass it explicitly if the auth is not needed.

Reply:
I'm a bit scared that someone can forget to pass it, but probably for now it's the better option, at least to reduce the changes introduced by the feature. Let's see if community contributors will agree.